### PR TITLE
Added Socials to the footer

### PR DESCRIPTION
**Pull request by aanand93**
Added the socials to the footer
Screen Shot 2021-11-15 at 1 42 39 PM

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -1,60 +1,91 @@
...
+                                About
+                            </Link>
+                        </li>
+                        <li>FAQ</li>
+                        <li>Help</li>
+                        <li>Terms of Use</li>
+                        <li>
+                            <Link to={"/blog"} style={link_style}>
+                                Blog
+                            </Link>
```
Comment (bcantello) [inline src/components/footer/footer.js:67]: collapse Link tag

**Review by bcantello — CHANGES_REQUESTED** [review]

File: src/components/footer/footer.js [hunk 2]
```diff
@@ -1,60 +1,91 @@
...
+                        </li>
+                    </ul>
```
[the remark this answers is not in the document]
Comment (bcantello) [reply src/components/footer/footer.js:69]: This list also needs to be moved farther to the right. The line at the bottom extends further to the right of the list content than it does to the left of the search bar. Ideally the line should terminate at the end of both. Ultimately this is more a function of the width of the right container div. The container extends too far to the right of its content.

File: src/components/footer/footer.js [hunk 3]
```diff
@@ -1,60 +1,91 @@
...
+import {
+    AiOutlineLinkedin,
+    AiOutlineInstagram,
+    AiOutlineTwitter,
+} from "react-icons/ai";
 
 /**
  * @returns {JSX.Element}
  * @constructor
  */
```
Comment (bcantello) [inline src/components/footer/footer.js:15]: Remove whitespace

File: src/components/footer/footer.js [hunk 4]
```diff
@@ -1,60 +1,91 @@
-import React, {useState} from "react";
+import React, { useState } from "react";
```
Comment (bcantello) [inline src/components/footer/footer.js:1]: This entire file, and all others that were edited, need to be reformatted to use tabs set to an indent of 4 instead of spaces. (This was stated as part of our style guide in the very long pinned slack post in dev. Understandable if you missed it at the end)

File: src/components/footer/footer.js [hunk 5]
```diff
@@ -1,60 +1,91 @@
...
+                        <AiOutlineLinkedin />
+                    </a>
+                    {/* <a href="https://www.instagram.com/{Spacelab}"> */}
+                    <AiOutlineInstagram />
+                    {/* </a> */}
+                    {/* <a href="https://twitter.com/{SpaceLab}"> */}
+                    <AiOutlineTwitter />
+                    {/* </a> */}
+                </IconContext.Provider>
+                <h5>@2021 by SpaceLab</h5>
```
[the remark this answers is not in the document]
Comment (bcantello) [reply src/components/footer/footer.js:87]: I think there is some extra padding or margin that is necessary on the top and bottom as well

File: src/components/footer/footer.js [hunk 6]
```diff
@@ -1,60 +1,91 @@
...
+                            <Link to={"/blog"} style={link_style}>
+                                Blog
+                            </Link>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div className="socialmedia-container">
+                <center>
+                    <hr width="80%" />
```
[the remark this answers is not in the document]
Comment (bcantello) [reply src/components/footer/footer.js:74]: additional margin is needed on the bottom for mobile and tablet

[the remark this answers is not in the document]
Comment (bcantello) [reply src/components/footer/footer.js:74]: For tablet: Some adjustments need to be made to overall content width allowances. The line length will need to be adjusted accordingly. Currently the content box is a little too wide, and the line comes in not quite wide enough.

**Review by bcantello — CHANGES_REQUESTED** [review]

File: src/components/footer/footer.js (hunk 4 above)
Comment (aanand93) [reply src/components/footer/footer.js:1]: For some reason I cannot figure out how to get the code to refactor on this component. I have changed the tab indent to 4, but it is still double the width. I have added all other changes and am about to push them

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (bcantello) [inline .vscode/settings.json:1]: This file should not be part of the pull request. Please remove and add this to your .gitignore file

**Review by bcantello — APPROVED** [review]
